### ISSUE TITLE
Cannot find credentials file for BigQuery method: service-account

### ISSUE THREAD

**Issue by ZeRego** [lightdash/lightdash#93]
**Summary:**
When using BigQuery service-account method [0] the app throws an error because it cannot find the credentials file.

![Screenshot 2021-06-18 at 16 57 31](https://user-images.githubusercontent.com/9117144/122590649-c161a200-d059-11eb-8fa8-93b487f93b26.png)

**Reproduction steps:**
Precondition: dbt project with a profile using BigQuery service-account method
Run lightdash (via docker)
```
# Go to your project
cd /path/to/my/dbt/project

export DBT_PROJECT_DIR=${PWD}
export DBT_PROFILES_DIR=${HOME}/.dbt
export LIGHTDASH_PORT=8080

docker run -p "${LIGHTDASH_PORT}:8080" -v "${DBT_PROJECT_DIR}:/usr/app/dbt" -v "${DBT_PROFILES_DIR}:/usr/app/profiles" lightdash/lightdash
```

[0] [https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile#service-account-file](https://docs.getdbt.com/reference/warehouse-profiles/bigquery-profile#service-account-file)

**Comment by owlas** [comment]
Hey @ZeRego - thanks! I don't think we've made this obvious enough.

Hidden in the docs here: https://docs.lightdash.com/get-started/setup-an-existing-dbt-project#for-bigquery-users

We have a snippet for copying in the gcloud config needed for oauth to work successfully.

Where did you find the snippet you posted above?

**Comment by kning** [comment]
I'm having this same issue and the docs link doesn't seem to exist anymore. Mind sharing a new one?

**Comment by owlas** [comment]
Hey @kning sorry about that! Here's the new link: https://docs.lightdash.com/get-started/setup-a-local-dbt-project#for-bigquery-users

**Comment by owlas** [comment]
Are you getting an error in the UI or only in the console?

**Comment by ZeRego** [comment]
We just need to create a docker volume for the key file.

@kning Could you try this ?

```
export DBT_PROJECT_DIR=${HOME}/Documents/jaffle_shop
export DBT_PROFILES_DIR=${HOME}/.dbt
export LIGHTDASH_PORT=8080
export KEY_FILE_PATH = /path/to/file.json # the same path you have in the profiles.yml

docker run -p "${LIGHTDASH_PORT}:8080" -v "${DBT_PROJECT_DIR}:/usr/app/dbt" -v 
"${DBT_PROFILES_DIR}:/usr/app/profiles" -v "${KEY_FILE_PATH}:${KEY_FILE_PATH}" lightdash/lightdash
```

Going to update the docs once someone confirms this works for them.

**Comment by kning** [comment]
Can confirm this worked, thanks! 

Note, there's a `\` missing in the docker command after the first line for the line break. Might want to add that in so it can copy/paste correctly out of the box especially if this is public facing.

**Comment by ZeRego** [comment]
Thanks for confirming the fix and noticing the typos.  
Just updated the comment and creating a PR to update the docs.

**Comment by owlas** [comment]
Closed by #255
